prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -977,6 +1007,8 @@ ssize_t rxr_pkt_proc_matched_rtm(struct rxr_ep *ep,
 		dlist_insert_tail(&rx_entry->rx_pending_entry, &ep->rx_pending_list);
 		ep->rx_pending++;
 #endif
+		if (rxr_get_rtm_base_hdr(pkt_entry->pkt)->flags & RXR_REQ_WAIT_RECEIPT)
+			rx_entry->fi_flags |= RXR_DELIVERY_COMPLETE_REQUESTED;
```

Comment:
maybe move this up, so this flag is set for all DC RTM.

Reply:
Ok.

Reply:
Moved up. Mark as resolved.